src/ad/azext_ad/generated/_help.py

```diff
@@ -0,0 +1,148 @@
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['ad ds'] = """
+    type: group
+    short-summary: Manage domain service with ad
+"""
+
+helps['ad ds list'] = """
+    type: command
+    short-summary: "The List Domain Services in Resource Group operation lists all the domain services available under \
+the given resource group. And The List Domain Services in Subscription operation lists all the domain services \
+available under the given subscription (and across all resource groups within that subscription)."
+    examples:
+      - name: List Domain Service By Group
+        text: |-
+               az ad ds list --resource-group "TestResourceGroup"
+      - name: List Domain Service By Sub
+        text: |-
+               az ad ds list
+"""
+
+helps['ad ds show'] = """
+    type: command
+    short-summary: "The Get Domain Service operation retrieves a json representation of the Domain Service."
+    examples:
+      - name: Get Domain Service
+        text: |-
+               az ad ds show --name "TestDomainService.com" --resource-group "TestResourceGroup"
+"""
+
+helps['ad ds create'] = """
+    type: command
+    short-summary: "The Create Domain Service operation creates a new domain service with the specified parameters. If \
+the specific service already exists, then any patchable properties will be updated and any immutable properties will \
+remain unchanged."
+    parameters:
+      - name: --replica-sets
+        short-summary: "List of ReplicaSets"
+        long-summary: |
+            Usage: --replica-sets location=XX subnet-id=XX
+
+            location: Virtual network location
+            subnet-id: The name of the virtual network that Domain Services will be deployed on. The id of the subnet \
+that Domain Services will be deployed on. /virtualNetwork/vnetName/subnets/subnetName.
+
+            Multiple actions can be specified by using more than one --replica-sets argument.
+      - name: --settings
+        short-summary: "List of settings for Resource Forest"
+        long-summary: |
+            Usage: --settings trusted-domain-fqdn=XX trust-direction=XX friendly-name=XX remote-dns-ips=XX \
+trust-password=XX
+
+            trusted-domain-fqdn: Trusted Domain FQDN
+            trust-direction: Trust Direction
+            friendly-name: Friendly Name
+            remote-dns-ips: Remote Dns ips
+            trust-password: Trust Password
+
+            Multiple actions can be specified by using more than one --settings argument.
+    examples:
+      - name: Create Domain Service
+        text: |-
+               az ad ds create --domain "TestDomainService.com" --ntlm-v1 "Enabled" --sync-ntlm-pwd "Enabled" --tls-v1 \
```

Comment:
with such complicated command, user is not easy to use.
We could consider a better interface design.

Reply:
This example sets almost all the properties which may make the command look complicated. I added example with required arguments and example with separated properties, see `manual/_help.py`